### PR TITLE
Disallow private branches for non-scratch container builds

### PR DESCRIPTION
**Pull request by bfontecc007**
This change will call callbacks with SCM info. If the build tag starts with rhel or RHEL, it will enforce scm policy. This has to be enabled by installing the scmpolicy plugin in koji build plugins. Under these conditions, branches starting with private are disallowed. 

Signed-off-by: Bret Fontecchio <bfontecc@redhat.com>

### CODE REVIEW

File: koji_containerbuild/plugins/builder_containerbuild.py [hunk 1]
```diff
@@ -33,6 +33,7 @@
 import signal
 
 import koji
+import koji.plugin
```
Comment (lcarva) [inline koji_containerbuild/plugins/builder_containerbuild.py]: Let's add a comment explaining this import (https://pagure.io/koji/pull-request/693)

File: tests/test_kcb.py [hunk 2]
```diff
@@ -283,6 +283,63 @@ def test_osbs_build(self, tmpdir, pkg_info, failure, orchestrator):
...
+        last_event_id = 456
+        koji_build_id = 999
+
+        options = flexmock(allowed_scms='pkgs.example.com:/*:no')
+
+        pkg_info = {'blocked': False}
+        session = flexmock()
+        (session
+            .should_receive('getLastEvent')
+            .and_return({'id': last_event_id}))
```
Comment (lcarva) [inline tests/test_kcb.py]: Let's make sure we add `.once()` where necessary in these expectations.

Comment (bfontecc007) [reply tests/test_kcb.py]: I added .times(2) to the callback check.

**Comment by lcarva** [comment]
Updated release notes draft:
> - `preSCMCheckout` and `postSCMCheckout` callbacks are triggered

**Pull request merged**